lib/OpenQA/Utils.pm

```diff
@@ -399,5 +400,15 @@ sub check_download_whitelist {
     return ();
 }
 
+sub notify_workers {
+    my $ipc = OpenQA::IPC->ipc;
+
+    my $con = $ipc->{bus}->get_connection;
+    my $msg = $con->make_method_call_message("org.opensuse.openqa.Scheduler", "/Scheduler", "org.opensuse.openqa.Scheduler", "emit_jobs_available");
```

Comment:
Is the introspection really querying remote object? I thought this info was registered in DBus daemon.